drivers/watchdog/Kconfig

```diff
@@ -13,12 +13,22 @@ if WATCHDOG
 
 config HAS_WDT_DISABLE_AT_BOOT
 	bool
+	help
+	  This hidden symbol must be selected by drivers which
+	  implement support for a watchdog timer that is enabled
+	  by default after system reset.
 
 config WDT_DISABLE_AT_BOOT
-	bool "Disable at boot"
+	bool "Disable watchdog timers at boot"
 	depends on HAS_WDT_DISABLE_AT_BOOT
 	help
-	  Disable watchdog at Zephyr system startup.
+	  Disable watchdog timers during Zephyr system startup.
```

Comment:
Disable watchdog timers *"during Zephyr system startup"* or *"at Zephyr system startup"*?
*"during"* could mean disable while system is starting up then enable them back.